pkg/patch/transformer/patchjson6902json.go

```diff
@@ -55,7 +70,7 @@ func (t *patchJson6902JSONTransformer) Transform(m resmap.ResMap) error {
 	}
 	modifiedObj, err := t.patch.Apply(rawObj)
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to apply json patch\n%s\n%v", string(t.rawOp), err)
```

Comment:
return nil, errors.Wrapf(err, "failed to apply JSON patch '%s'", string(t.rawOp))

Reply:
updated